front/app/services/projects.ts

```diff
@@ -246,10 +245,8 @@ export interface IProjectFormState {
   apiErrors: { [fieldName: string]: CLError[] };
   saved: boolean;
   areas: IAreaData[];
-  areaType: 'all' | 'selection';
   locale: Locale;
   currentTenant: IAppConfiguration | null;
-  areasOptions: IOption[];
   submitState: ISubmitState;
```

Comment:
Not needed anymore in the FormState, since these moved to the GeographicAreas component.